packages/models-library/tests/image-meta.yaml

```diff
@@ -0,0 +1,51 @@
+name: sleeper
+key: simcore/services/comp/itis/sleeper
+type: computational
+integration-version: 1.0.0
+version: 2.0.2
+description: A service which awaits for time to pass.
+contact: neagu@itis.swiss
+authors:
+  - name: "Manuel Guidon"
+    email: guidon@itis.swiss
+    affiliation: "IT'IS Foundation"
+  - name: "Odei Maiz"
+    email: maiz@itis.swiss
+    affiliation: "IT'IS Foundation"
+  - name: "Andrei Neagu"
+    email: neagu@itis.swiss
+    affiliation: "IT'IS Foundation"
+inputs:
+  input_1:
+    displayOrder: 1
+    label: File with int number
+    description: Pick a file containing only one integer
+    type: "data:text/plain"
+    fileToKeyMap:
+      single_number.txt: input_1
+  input_2:
+    displayOrder: 2
+    label: Sleep interval
+    description: Choose an amount of time to sleep
+    type: integer
+    defaultValue: 2
```

Comment:
defaultValue is optional, maybe worth to have one case without it?

Reply:
``input_1`` has already that case

Reply:
no it's a file